src/lang/en.json

```diff
@@ -0,0 +1,391 @@
+          },
+          "operationId": {
+            "label": "Operation ID"
+          },
+          "summary": {
+            "label": "Summary"
+          },
+          "description": {
+            "label": "Description"
+          },
+          "deprecated": {
+            "label": "Deprecated?",
+            "choices": {
+              "blank": ""
+            }
+          },
+          "overrides": {
+            "label": "Overrides",
+            "schemes": {
+              "label": "Protocols"
+            }
+          },
+          "tags": {
+            "label": "Tags",
+            "newItemText": "New Tag",
+            "item": {
+              "label": "Tag #$index"
+            }
+          },
+          "responses": {
+            "label": "Responses",
+            "newItemText": "New Response"
+          },
+          "parameters": {
+            "label": "Params",
+            "newItemText": "New Parameter"
+          }
+        },
+        "reference": {
+          "label": "Target"
+        }
+      }
+    },
+    "global-security": {
+      "label": "Security",
+      "definitions": {
+        "label": "Definitions",
+        "newItemText": "New Security Definition",
+        "item": {
+          "label": "Security Definition #$index",
+          "basic": {
+            "description": {
+              "label": "Description"
+            }
+          },
+          "apiKey": {
+            "description": {
+              "label": "Description"
+            },
+            "name": {
+              "label": "Name"
+            },
+            "in": {
+              "label": "API key location",
+              "choices": {
+                "query": "query",
+                "header": "header"
+              }
+            }
+          },
+          "oauth2": {
+            "description": {
+              "label": "Description"
+            },
+            "flow": {
+              "label": "Flow",
+              "choices": {
+                "implicit": "implicit",
+                "password": "password",
+                "application": "application",
+                "accessCode": "accessCode"
+              }
+            },
+            "authorizationUrl": {
+              "label": "Authorization URL"
+            },
+            "scopes": {
+              "label": "Scopes",
+              "newItemText": "New Scope",
+              "item": {
+                "label": "Scope #$index",
+                "key": {
+                  "label": "Key"
+                },
+                "description": {
+                  "label": "Description"
+                }
+              }
+            }
+          }
+        }
+      },
+      "requirements": {
+        "label": "Requirements",
+        "newItemText": "New Requirements Object",
+        "item": {
+          "label": "Requirements Object #$index",
+          "newItemText": "New Security Requirement",
+          "item": {
+            "label": "Security Requirement #$index",
```

Comment:
My intuition here is telling me that we might only use i18next [interpolation](https://www.i18next.com/interpolation.html) and [pluralization](https://www.i18next.com/plurals.html), leaving most/all other parameters and markup out of the localization strings.